lib/parser.js

```diff
@@ -137,8 +137,8 @@ module.exports = (function() {
         peg$c90 = { type: "class", value: "[-+*=]", description: "[-+*=]" },
         peg$c91 = /^[\/|]/,
         peg$c92 = { type: "class", value: "[\\/|]", description: "[\\/|]" },
-        peg$c93 = /^[\-0-9a-zA-Z+*,=():\/;?.!'` \x80-\uD7FF\uE000-\uFFFF]/,
-        peg$c94 = { type: "class", value: "[-0-9a-zA-Z+*,=():\\/;?.!\\'` \\x80-\\ud7ff\\ue000-\\uffff]", description: "[-0-9a-zA-Z+*,=():\\/;?.!\\'` \\x80-\\ud7ff\\ue000-\\uffff]" },
+        peg$c93 = /^[\-0-9a-zA-Z+*,=():\/;?.!'` [\]\x80-\uD7FF\uE000-\uFFFF]/,
```

Comment:
I see. My suggestion was for consistency's sake, but you are right, let's note change generated code.